test/support/constants/test-currency.constant.ts

```diff
@@ -1,4 +1,4 @@
 export const TEST_CURRENCIES = {
   A_TEST_CURRENCY: 'EUR',
-  NON_GBP_CURRENCY: 'USD',
+  NON_GBP_CURRENCY: 'AUD',
```

Comment:
Is `USD` not applicable?

Reply:
Yes it is, I just changed it from `USD` as there is a separate case for USD repayment schedules